### PR TITLE
Use mystified certs

### PR DESCRIPTION
**Pull request by knuton**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "start": "nodemon src/server.js",
     "electron": "electron .",
-    "build": "yarn && npm run check-cert && rimraf build && mkdir -p build && node bin/build.js && node bin/package.js",
+    "build": "yarn && npm run check-cert && rimraf build && mkdirp build && node bin/build.js && node bin/package.js",
```
Comment (pukkamustard) [inline package.json]: Is `mkdirp` provided by node or is installed as a package? (I don't have mkdirp on my system)

Comment (pukkamustard) [reply package.json]: Nevermind, it seems to work. But don't know where mkdirp comes from...

Comment (knuton) [reply package.json]: Yup, `node_modules`.

**Comment by pukkamustard** [comment]
[Investigate Certificate revocation error for localhost.dividat.com](https://trello.com/c/E0zYk20h/78-investigate-certificate-revocation-error-for-localhostdividatcom)

**Pull request merged**